manual/storage.md

```diff
@@ -0,0 +1,72 @@
+# Storage
+
+The first step in interacting with storage is to create a Rune representing the
+specific piece of storage.
+
+## Storage Types
+
+Let's use the fluent root to access some different kinds of storage.
+
+> Note: the following examples assume the same imports:
```

Comment:
I don't think this note is necessary/useful. Also, they assume `polkadotDev` as well as `polkadot`